### PR TITLE
README.md created

### PR DESCRIPTION
**Pull request by vlebedev**

### CODE REVIEW

**Comment by rtsisyk** [comment]
Text is OK. It would be nice to add logos, badges and so on, but it can be done later by further pull requests.

File: README.md [hunk 1]
```diff
@@ -1 +1,45 @@
-# stegos
\ No newline at end of file
+# Stegos Blockchain
```
Comment (emtq) [inline README.md]: I would remove this section. We should have a LICENSE file so the license should be obvious. I copied the body of the README to Dropbox Paper to edit it.

Comment (emtq) [reply README.md]: Yes, please remove the section.

File: README.md [hunk 2]
```diff
@@ -1 +1,45 @@
...
+
+This source tree constitutes the public Stegos blockchain source repository from https://github.com/stegos/stegos.
+
+We intend to do our work transparently and openly in full public view to this repository.
+
+We assert an MIT license over this source aggregation; see https://github.com/stegos/stegos/blob/dev/LICENSE.
+
+## What is Stegos?
+
```
Comment (emtq) [inline README.md]: Please replace the content of this section with https://paper.dropbox.com/doc/Github-README--AOaLGsZyp2LfhCSxZ80sdLRtAQ-jC9Lukf9yTb5P80CGXIWp

File: README.md [hunk 3]
```diff
@@ -1 +1,45 @@
...
+The Jackpot is an unique feature of Stegos. A fraction of the fees from each block and all slashed stakes of validators caught cheating go to the Jackpot. Each few thousand blocks, validators run cryptographic lottery based on verifiable distributed randomness, and transfer the contents of the Jackpot to the lottery winner. The longer a validator keeps it stake and participates in the consensus, the higher probability it has to win the Jackpot lottery.
+
+## Project Status
+
+We are in the very early days of the development. We are aiming to build our blockchain iteratively, using two-week cycles (spints). At the end of each sprint we will deliver something working and testable.
```
Comment (emtq) [inline README.md]: Remove these 2 sentences, up to the links.

File: README.md [hunk 4]
```diff
@@ -1 +1,45 @@
...
+
+Issue tracker is at https://github.com/stegos/stegos/issues.
+Project boards are at https://github.com/stegos/stegos/projects.
+
+As our code, our issue tracker and project boards are completely open. We have nothing to hide from the public view.
```
Comment (emtq) [inline README.md]: Remove this sentence.

File: README.md [hunk 5]
```diff
@@ -1 +1,45 @@
...
+We are in the very early days of the development. We are aiming to build our blockchain iteratively, using two-week cycles (spints). At the end of each sprint we will deliver something working and testable.
+
+Issue tracker is at https://github.com/stegos/stegos/issues.
+Project boards are at https://github.com/stegos/stegos/projects.
+
+As our code, our issue tracker and project boards are completely open. We have nothing to hide from the public view. 
+
+## How To Contribute
+
+Even one word corrections are welcome! Our objective is to encourage you to get interested in Stegos and contribute in any way possible.
```
Comment (emtq) [inline README.md]: Replace the 2 sentences with "We encourage you to contribute in any way you can.

**Pull request merged**